__tests__/backend/utils/colorTemperature.test.js

```diff
@@ -3,9 +3,13 @@
  */
 
 import 'react-native';
-import React from 'react';
-import {getColorTempGradient} from '../../../src/utils/colorTemperature';
+import {getColorTempGradient} from '../../../src/helpers/colorTemperature';
+
 // Note: test renderer must be required after react-native.
-import renderer from 'react-test-renderer';
+//import renderer from 'react-test-renderer';
 
-test.skip('getColorTempGradient', () => {});
+test('getColorTempGradient', () => {
+  for (var i = -20; i < 121; i + 20) {
```

Comment:
Ok we gotta review testing strategies. 

Read this:
https://dev.to/bgord/simplify-repetitive-jest-test-cases-with-test-each-310m

As well as:

https://jestjs.io/docs/en/using-matchers

https://jestjs.io/docs/en/expect